src/main/java/com/atlassian/plugins/confluence/markdown/MarkdownFromURLMacro.java

```diff
@@ -228,6 +229,20 @@ public String execute(Map<String, String> parameters, String bodyContent, Conver
 					".hljs {display: inline;}\n" +
 					"pre > code {display: block !important;}\n" +
 					"</style>";
+                    
+            String rendermermaidjs ="<script>\n"+
+                "var config = {\n"+
+                "    startOnLoad:true,\n"+
+                "    theme: 'default',\n"+
+                "    flowchart:{\n"+
+                "            useMaxWidth:false,\n"+
+                "            htmlLabels:true\n"+
+                "        }\n"+
+                "};\n"+
+                "mermaid.initialize(config);\n"+
+                "window.mermaid.init(undefined, document.querySelectorAll('.language-mermaid'));\n"+
```

Comment:
``.language-mermaid`` could to be to broad and might include elements from other macros. Let's use something like this to limit the scope to just MardownMacro:
```
[data-macro-name=\"markdown-from-url\"] .language-mermaid'
```